### PR TITLE
Refactor cognito script

### PR DESCRIPTION
**Pull request by daonhs**

### CODE REVIEW

**Review by ArifNhs — DISMISSED** [review]
Approved

**Review by stefaniuk — DISMISSED** [review]
Yes, looks good. Thanks Daniel

**Review by jope1 — DISMISSED** [review]

**Review by ArifNhs — DISMISSED** [review]
Approved

File: infrastructure/modules/cognito/secrets.tf [hunk 1]
```diff
@@ -7,7 +7,7 @@ resource "aws_secretsmanager_secret" "cognito_admin_password" {
 
 resource "aws_secretsmanager_secret_version" "cognito_admin_password" {
   secret_id     = aws_secretsmanager_secret.cognito_admin_password.id
-  secret_string = random_password.cognito_admin_password.result
+  secret_string = jsonencode({ "password" : "$random_password.cognito_admin_password.result" })
```
Comment (gwaardenburg) [inline infrastructure/modules/cognito/secrets.tf:10]: Does this mean that all the users are going to have the same password?

**Review by gwaardenburg — APPROVED** [review]

**Review by ArifNhs — APPROVED** [review]
Approved

**Pull request merged**